templatestore/frontend/src/pages/templateScreen.js

```diff
@@ -153,41 +161,87 @@ class TemplateScreen extends Component {
                         config: response.data
                     });
                 }
+                this.updateurlKeyList();
             })
             .catch(function(error) {
                 console.log(error);
             });
+        this.settinyUrlObj();
+    }
+
+    scan(parent, obj, result) {
+        var k;
+        if (obj instanceof Object) {
+            for (k in obj) {
```

Comment:
instead of for-in, try using Object.keys() and iterate over that.
Using that, you wont have to check for hasOwnProperty